### DISCUSSION TITLE
Where do we store code that runs every X seconds?

### DISCUSSION THREAD

**Discussion by Nicholas705M** [Heroic-Games-Launcher/HeroicGamesLauncher#2926] [Q&A]
Where do we store code that runs every X seconds (the context is I want to add code that refreshes the library every 60 seconds)? Do we have a specific file for that? Are there design docs that say where we should put our code in general?

**Comment by arielj** [comment]
We don't do that anywhere so there's no codes nor a specific place

you probably need to start a js timer in the backend when the app starts that does that

but I'd advise against it, unless users buy games super often this is just going to be a waste of resources updating the library for nothing most of the times
